chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -103,6 +103,9 @@ const REPORT_BANDWIDTH_THRESHOLD_COUNT: usize = 10_000;
 /// How long a peer has to be unreachable, until we prune it from the in-memory graph.
 const PRUNE_UNREACHABLE_PEERS_AFTER: time::Duration = time::Duration::hours(1);
 
+// If a peer is more than these blocks behind (comparing to our current head) - don't route any messages through it.
+const UNRELIABLE_PEER_HORIZON: u64 = 60;
```

Comment:
How is this number chosen?

Reply:
(also added a comment):
to roughly match the period in which we re-evaluate which peers are unreliable (which we do every minute).
We didn't want to set it too low (as it might move many peers to unreliable, and cause routing issues).